debug_toolbar/static/debug_toolbar/js/toolbar.profiling.js

```diff
@@ -0,0 +1,3 @@
+(function () {
+    djdt.applyStyle('padding-left');
+})();
```

Comment:
I think I'd be more in favor of an approach that would find all elements that have a class such as `djdt-dynamic-style`, then have it apply the styles listed in the data attribute. That way the javascript doesn't need to know anything about the specific styles that the HTML elements are having applied.